src/serac/physics/base_physics.cpp

```diff
@@ -72,20 +77,17 @@ void BasePhysics::initializeOutput(const serac::OutputType output_type, const st
       break;
     }
 
-    case serac::OutputType::GLVis: {
-      break;
-    }
-
+    case serac::OutputType::GLVis:
+      [[fallthrough]];
     case OutputType::SidreVisIt: {
-      dc_ = std::make_unique<axom::sidre::MFEMSidreDataCollection>(root_name_, &state_.front().get().mesh());
       break;
     }
 
     default:
       SLIC_ERROR_ROOT("OutputType not recognized!");
   }
 
-  if ((output_type_ == OutputType::VisIt) || (output_type_ == OutputType::SidreVisIt)) {
+  if (output_type_ == OutputType::VisIt) {
```

Comment:
Can you add some comments explaining this function? I'm sure there is a reason but it seems weird there is two case statements of basically the same thing.